src/main/java/frc/robot/Subsystems/IntakeToTransfer.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.RobotMap;
+
+/**
+ * Uses a wheel located between the transfer and intake subsystem to hold/move cargo pieces.
+ */
+public class IntakeToTransfer extends MotoredGenericSubsystem {
+
+    public static final double SPEED = 0.7;
+
+    private static IntakeToTransfer instance;
+
+    private final DigitalInput limit;
+
+    public static IntakeToTransfer getInstance() {
+        if (instance == null) {
+            instance = new IntakeToTransfer(new WPI_VictorSPX(RobotMap.CAN.INTAKE_TO_TRANSFER_VICTOR));
+        }
+        return instance;
+    }
+
+    private IntakeToTransfer(WPI_VictorSPX victor) {
```

Comment:
Do you need this to be a victor? If they change it to talon why change this in 2 places? 
True for all subsystems